views/forms/BarChartFields.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from 'react'
+import Delete from '@material-ui/icons/Delete';
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography';
+
+const title = 'title'
+
+const BarChartFields = ({ 
+  classes,
+  updateFormValues,
+  formValues,
+  fieldLabelValueMap,
+  addValueAndLabelField,
+  removeField,
+  updateFieldValues
+}) => {
+  const [chartTitle, setTitle] = useState('')
+  
+  useEffect(() => {
+    setTitle(formValues[title])
+  }, [formValues, fieldLabelValueMap])
+
+  return(
+    <>
+      <Typography variant='subtitle1' gutterBottom>
+        {chartTitle} 
+      </Typography>
+      <TextField
+        className={classes.textInput}
+        label='Title'
+        name='title'
+        onChange={updateFormValues}
+        required
+        fullWidth
+      />
+      <TextField
+        className={classes.textInput}
+        label='Assessment'
+        name='assessment'
+        onChange={updateFormValues}
+        required
+        fullWidth
+      />
+      <TextField
+        label='Variable Name'
+        name='variable'
+        className={classes.textInput}
+        onChange={updateFormValues}
+        required
+        fullWidth
+      />
+      {
+        fieldLabelValueMap.length > 0 && 
+        fieldLabelValueMap.map((_, idx) => (
+          <div key={idx} className={classes.formLabelRow}>
+            <TextField
+              label='Value'
+              name='value'
+              onChange={(e) => updateFieldValues(e, idx)}
+              className={`
+                ${classes.formLabelCol} 
+                ${classes.variableListInput}
+              `}
+              required
```

Comment:
I think we should also set the `value` prop of the text field since react prefers that vs allowing the browser to control it.

Reply:
`js
  <TextInput 
    value />

// OR

 <TextInput
  value={''}
 />
`

Reply:
[78b1245](https://github.com/AMP-SCZ/dpdash/pull/84/commits/78b12458361f85aeb0919c885be4ca8c8f6c047b)